virt/live_migration/virt-migrate-vmi.adoc

```diff
@@ -9,6 +9,11 @@ toc::[]
 
 Manually initiate a live migration of a virtual machine instance to another node using either the web console or the CLI.
 
+[NOTE]
+====
+If a virtual machine uses a host model CPU, you can perform live migration of that virtual machine between nodes that support that CPU model. Migration is not allowed to nodes that do not support the virtual machine's CPU model.
```

Comment:
s/perform live migration of that/live migrate the

Reply:
I really prefer the noun version "live migration" to the verbal expression "live migrate." I know that the noun version uses an extra word"perform live migration" vs "live migrate" but I would imagine it is not a deal breaker here? Since this is a suggestion, I am leaving the content as it is with the explanation I've provided.